scripts/cybershake/run_cybershake.py

```diff
@@ -243,6 +243,8 @@ def main():
     parser.add_argument(
         "config_file",
         help="The location of the config file containing everything to be run",
+        nargs="?",
+        default=join(dirname(realpath(__file__)), "task_config.yaml"),
```

Comment:
The file should probably be kept in the templates directory, cybershake is getting a bit cluttered.